src/runtime_src/core/edge/drm/zocl/zocl_drv.c

```diff
@@ -863,6 +863,19 @@ static int zocl_drm_platform_probe(struct platform_device *pdev)
 	}
 	mutex_init(&zdev->mm_lock);
 
+#ifdef CONFIG_ARCH_DMA_ADDR_T_64BIT
+	/* Platform did not initialize dma_mask, try to set 64-bit DMA first */
+	ret = dma_set_mask_and_coherent(&pdev->dev, DMA_BIT_MASK(64));
+	if (ret) {
+		/* If seting 64-bit DMA mask fails, fall back to 32-bit DMA mask */
+		ret = dma_set_mask_and_coherent(&pdev->dev, DMA_BIT_MASK(32));
+		if (ret) {
+			dev_err(&pdev->dev,"DMA configuration failed: 0x%x\n", ret);
```

Comment:
You may want to use DRM_ERR instead of dev_err

Reply:
Done